geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/fd/GMSHealthMonitor.java

```diff
@@ -1277,6 +1277,9 @@ private boolean inlineCheckIfAvailable(final InternalDistributedMember initiator
     boolean failed = false;
 
     membersInFinalCheck.add(mbr);
+    suspectedMemberInView.putIfAbsent(mbr, this.currentView);
+    setNextNeighbor(GMSHealthMonitor.this.currentView, mbr);
```

Comment:
How does `GMSHealthMonitor.this.currentView` differ from `this.currentView`?

Reply:
I copied that code from the Monitor class.  I'll remove GMSHealthMonitor.this.